source/_integrations/calendar.markdown

```diff
@@ -2,6 +2,7 @@
 title: Calendar
 description: Instructions on how to integrate calendars within Home Assistant.
 ha_release: 0.33
+ha_iot_class: Local Push
```

Comment:
```suggestion
ha_iot_class: ~
```

This integration does work by itself. It is used by other integrations to build upon. Hence, it doesn't an IoT class really. Let's make it ~ (= null/none) for now.